.python-version

```diff
@@ -1 +1 @@
-3.8.16
+3.9.17
```

Comment:
missing changes in doc/dependencies.md ?

Reply:
bump @PastaPastaPasta